pycbc/events/stat.py

```diff
@@ -1546,7 +1543,8 @@ def coinc_lim_for_thresh(self, s, thresh, limifo,
         """
 
         # Safety against subclassing and not rethinking this
-        allowed_names = ['ExpFitSGFgBgNormStatistic']
+        allowed_names = ['ExpFitSGFgBgNormStatistic',
+                         'ExpFitSGPSDFgBgNormBBHStatistic']
```

Comment:
I'll have to take a bit more time to look through this, but this looks like PSD and BBH are mutually required, is that correct/desired?

Reply:
It is not actually correct anymore, the names are somewhat misleading atm. Would you suggest renaming? I could do that.

Reply:
i.e. you can use any single stat now there in either of the multi-det coinc classes

Reply:
In that case removing both "SG" and "PSD" should make sense

Reply:
OK, I can do that. I'll need to update the test config files as well. And you all should note that for your internal configs as well if  using these classes.

Reply:
@GarethDaviesGW Ok, I've renamed everything by dropping the sg / psd parts everywhere.